pnpm-lock.yaml

```diff
@@ -19106,7 +19106,7 @@ packages:
     engines: {node: ^14.15.0 || ^16.10.0 || >=18.0.0}
     dependencies:
       '@jest/types': 29.6.3
-      '@types/node': 18.19.1
+      '@types/node': 20.12.6
```

Comment:
We shouldn't have all the @types/node updates. Curious, what steps did you follow (e.g. which environment and which folder did you run `pnpm i` from) for this? I just tried the same update to `joi` locally and don't see the @types/node changes...

Reply:
I ran `pnpm install joi@^17.13.1` from /experimental/PropertyDDS/packages/property-query

Reply:
Interesting, locally I also get the @types/node changes doing that... can you try manually updating the version in package json and then just running `pnpm i` from the root of the repo?

Reply:
@seanimam Can you spare a few minutes to update this page in the wiki with anything relevant you learn from this exercise? https://github.com/microsoft/FluidFramework/wiki/Managing-dependencies

I'm trying to make that the "go-to" destination for "how to update deps in the repo" guides.

Reply:
I made a post in our internal Dev channel with my findings, you can use that as a starting point to update the wiki @seanimam

Reply:
Just to confirm, I used your method @alexvy86 and the lock file did not have the extraneous changes to node.